frontend/src/pages/cohorts/CohortsPage.tsx

```diff
@@ -6,8 +6,8 @@ import { Dispatch, SetStateAction, useState } from "react";
 import {
   MAX_ROWS_EXPORT,
   MAX_ROWS_EXPORT_WARNING,
-  WesSampleDetailsColumns,
-  CohortsListColumns,
+  WesSampleColumns,
+  CohortColumns,
```

Comment:
Renamed and simplified all column definition variable names.
1. `SampleMetadataDetailsColumns` is incorrect as it includes data beyond SampleMetadata
2. Names like `CohortsListColumns` reflect the old GraphQL query names (`query CohortsList` in this case)